src/gcp/storage.ts

```diff
@@ -143,19 +151,19 @@ export async function getDefaultBucket(projectId: string): Promise<string> {
   await ensure(projectId, "firebasestorage.googleapis.com", "storage", false);
   try {
     const localAPIClient = new Client({ urlPrefix: firebaseStorageOrigin, apiVersion: "v1alpha" });
-    const response = await localAPIClient.get<{ name: string }>(
+    const response = await localAPIClient.get<GetDefaultBucketResponse>(
       `/projects/${projectId}/defaultBucket`
     );
-    if (!response.body?.name) {
+    if (!response.body?.bucket.name) {
       logger.debug("Default storage bucket is undefined.");
       throw new FirebaseError(
         "Your project is being set up. Please wait a minute before deploying again."
       );
     }
-    return response.body.name;
+    return response.body.bucket.name.split("/").pop()!;
   } catch (err: any) {
     logger.info(
-      "\n\nThere was an issue deploying your Storage rules. Verify that your project has a Google App Engine instance setup at https://console.cloud.google.com/appengine and try again. If this issue persists, please contact support."
+      "\n\nThere was an issue deploying your functions. Verify that your project has a Google App Engine instance setup at https://console.cloud.google.com/appengine and try again. If this issue persists, please contact support."
```

Comment:
Quick sanity check - is this still accurate/ relevant? If someone doesn't have a default storage bucket set up, do they still need to go create an App engine instance, or is there something else they can do?

Reply:
As of now it's the correct path of action. We also have a new createDefaultBucket endpoint that we can ask clients to call which right now under the hood among other things will create an AppEngine instance. https://firebase.google.com/docs/reference/rest/storage/rest/v1alpha/projects.defaultBucket/create

I would say we change the copy once the endpoint hits v1beta as opposed to v1alpha